scripts/install-toolchain.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+PLATFORM=$(uname -s | tr '[:upper:]' '[:lower:]')
+ARCH=$([[ $(uname -m) = "x86_64" ]] && echo 'amd64' || echo 'arm64')
```

Comment:
```suggestion
ARCH=$(uname -m| sed 's/aarch64/arm64/' | sed 's/x86_64/amd64/')
```
If you wanted one less sub-shell and to avoid assuming non-amd64 = arm64.